Mage.Tests/src/test/java/org/mage/test/commander/duel/NorinTheWaryTest.java

```diff
@@ -83,8 +83,9 @@ public void castNorinTheWaryToCommandAndReturn() {
         execute();
 
         assertGraveyardCount(playerA, "Lightning Bolt", 1);
-        assertPermanentCount(playerA, "Norin the Wary", 1);
+        assertPermanentCount(playerA, "Norin the Wary", 0);
         assertExileCount("Norin the Wary", 0);
+        assertCommandZoneCount(playerA,"Norin the Wary",1);
```

Comment:
Quick nitpick: This test's name is no longer accurate, since the rules change means you can't flicker Norin to the command zone anymore.